core/src/builder.ts

```diff
@@ -84,39 +76,44 @@ function PropertyBuilder<Model extends Json>({
 }
 
 function Builder<Model extends Json>({
-  defaults,
   generator,
   transformers,
 }: TBuilderOptions<Model> = {}): TBuilder<Model> {
-  const propertyBuilder = PropertyBuilder<Model>({ defaults });
   const applyGeneratorIfExists = (): Partial<Model> => {
     if (!generator) return {};
-    return generator.generate({ defaults });
+    return generator.generate();
   };
 
+  const propertyBuilder = PropertyBuilder<Model>(applyGeneratorIfExists());
```

Comment:
So this is where we "generate things" before hand, correct?

Reply:
Yup.